src/index.html

```diff
@@ -4,9 +4,497 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a
+          href="./index.html"
+          class="header__logo-link"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo"
+            class="header__logo-img"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li>
+              <a href="#" class="nav__list-link is-active">Apple</a>
+            </li>
```

Comment:
check this checklist point
<img width="945" alt="image" src="https://user-images.githubusercontent.com/102411130/216546757-f0891472-f6c1-44f8-a3da-5c2c51db68f9.png">